src/content/developers/docs/scaling/plasma/index.md

```diff
@@ -18,9 +18,36 @@ You should have a good understanding of all the foundational topics and a high-l
 | High throughput, low cost per transaction.                                                                            | Does not support general computation. Only basic token transfers, swaps, and a few other transaction types are supported via predicate logic.                               |
-| Good for transactions between arbitrary users (no overhead per user pair if both are established on the plasma chain). | Need to periodically watch the network (liveness requirement) or delegate this responsibility to someone else to ensure the security of your funds.                         |
-|                                                                                                                       | Relies on one or more operators to store data and serve it upon request.                                                                                                    |
-|                                                                                                                       | Withdrawals are delayed by several days to allow for challenges. For fungible assets this can be mitigated by liquidity providers, but there is an associated capital cost. |
+| Good for transactions between arbitrary users (no overhead per user pair if both are established on the plasma chain) | Need to periodically watch the network (liveness requirement) or delegate this responsibility to someone else to ensure the security of your funds.                         |
+|  Plasma chains can be adapted to specific use-cases that are unrelated to the main chain. Anyone, including businesses, can customize Plasma smart contracts to provide scalable infrastructure that works in different contexts.                                                       | Relies on one or more operators to store data and serve it upon request.                                                                                              |
+|                                                                                                                        | Withdrawals are delayed by several days to allow for challenges. For fungible assets this can be mitigated by liquidity providers, but there is an associated capital cost. |
+|                                                                                                                        | If too many users try to exit their Plasma chain simultaneously (due to malicious activity or network failure), they could flood the root chain and congest the network. | 
+
+## How does Plasma differ from sidechains and sharding? 
+
+Plasma, sidechains, and sharding all follow a similar pattern where a collection of "child chains" rely on a "main chain" for consensus. At intervals, cryptographic proofs of transaction validity (hashes) are submitted to the main chain to prevent fraudulent activity on child chains. 
```

Comment:
> I was trying to show the baseline similarities between the thre

> Vitalik uses a "hub-and-spoke" model to describe all three

For sure... honestly a bit torn on this approach. I just hesitate because the spokes connecting a side chain to Mainnet are _significantly_ weaker than the spokes that will connect data shards to the Beacon Chain, or that connect plasma chains to Mainnet...

> only problem is that the consensus backbone part may not really apply to sidechains.

...to that point exactly. Just want to try and make that clear for readers, since misunderstanding these spokes can potentially lead to loss of funds. I think I would focus more on the differentiating factors here, while keeping any similarities high level.

Reply:
Yeah, so I changed the language to suggest that the only similarity between the three is that they connect to Ethereum Mainnet "in some way." I also stated that the level and strength of those connections vary, which can have implications for security. I think that keeps the similarities high-level enough, don't you think?

Reply:
> Vitalik uses a "hub-and-spoke" model to describe all three--that is, the main chain serves as a "consensus backbone" while the connected chains (plasma child chains, sidechains, and shard chain) handle actual transactions. We can use this mental model, but the only problem is that the consensus backbone part may not really apply to sidechains.

One thing to be cognizant of here is how fast Ethereum is moving. Terminology (and Vitalik's views, generally) from 2019, are probably dramatically different than they are now.